recipes/boost/all/conanfile.py

```diff
@@ -624,27 +371,21 @@ def _stacktrace_from_exception_available(self):
             # https://github.com/boostorg/stacktrace/blob/boost-1.85.0/build/Jamfile.v2#L143
             return not self.options.header_only and not self.options.without_stacktrace and self.settings.os != "Windows"
         elif Version(self.version) >= "1.86.0":
+            # https://github.com/boostorg/stacktrace/issues/196
+            if self.settings.os == "Windows" and self.settings.compiler == "gcc":
+                return False
             # https://github.com/boostorg/stacktrace/blob/boost-1.86.0/build/Jamfile.v2#L148
+            # FIXME: from_exception is enabled by for intel only, but still can be built as exposed property.
             return not self.options.header_only and not self.options.without_stacktrace and self._b2_architecture == "x86"
 
     def configure(self):
         if self.options.header_only:
             self.options.rm_safe("shared")
             self.options.rm_safe("fPIC")
+            self.package_type = "header-library"
```

Comment:
This should be automatic, not needed

Reply:
Fixed in the commit [da40105](https://github.com/conan-io/conan-center-index/pull/26030/commits/da401051381b0fc60b400651aee200ff214c3a76)